src/components/timeline/Day.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import styled from 'styled-components';
+import { colors } from 'common/colors';
+import { media } from 'common/layout';
+
+interface DayProps {
+  children: React.ReactNode;
+  day: string;
+}
+
+const Day = ({ children, day }: DayProps): JSX.Element => {
```

Comment:
We took a little cleanup of the eslint rules and decided to remove the requirement of defining `JSX.Element`.